src/main/features/testing-support/views/create-draft.njk

```diff
@@ -0,0 +1,21 @@
+{% extends "layout.njk" %}
+{% from "form.njk" import linkButton, csrfProtection, errorSummary %}
+{% from "date.njk" import dateInput %}
+
+{% set title = 'Create Draft | Testing support' %}
+{% set heading = t('Create Draft') %}
```

Comment:
@geek-soft Do we require translation for testing support pages i.e. t('Create Draft')

Reply:
Not really, testing support is just for us however I see we already started wrapping wording with `t` function on other testing support screens.

Reply:
Removed translation